Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Profile.hs

```diff
@@ -230,12 +232,17 @@ updateDefaultEmergencyNumbers personId req = do
   where
     buildPersonDefaultEmergencyNumber now defEmNum = do
       encMobNum <- encrypt defEmNum.mobileNumber
+      dbHash <- getDbHash defEmNum.mobileNumber
+      mbEmNumPerson <- QPerson.findByMobileNumberAndMerchantId defEmNum.mobileCountryCode dbHash merchantId
       return $
         DPDEN.PersonDefaultEmergencyNumber
           { mobileNumber = encMobNum,
             name = defEmNum.name,
             mobileCountryCode = defEmNum.mobileCountryCode,
             createdAt = now,
+            contactPersonId = (.id) <$> mbEmNumPerson,
+            isFollowing = True,
```

Comment:
users can't decide who they want to be notified and who not ?

Reply:
This value will be updated in another screen while adding. @prakharritik will update this value in his PR.